src/Testing/TypeInferenceTestCase.php

```diff
@@ -157,21 +162,32 @@ public static function gatherAssertTypes(string $file): array
 			$functionName = $nameNode->toString();
 			if (in_array(strtolower($functionName), ['asserttype', 'assertnativetype', 'assertvariablecertainty'], true)) {
 				self::fail(sprintf(
-					'Missing use statement for %s() on line %d.',
+					'Missing use statement for %s() in %s on line %d.',
 					$functionName,
+					$pathHelper->getRelativePath(strtr($file, '\\', '/')),
```

Comment:
We don't need to change the backslashes here I think. And if you wanted to, we have `FileHelper::normalizePath()` for that. You can run this function on both sides here (implementation & tests).

Reply:
the currentWorkingDirectory in https://github.com/phpstan/phpstan-src/blob/519982c55e43a20f72b02316e47327dfb1ba702b/src/File/SimpleRelativePathHelper.php#L19 is normalized to forward slashes.

when not doing `strtr($file, '\\', '/')` we pass on windows a path with backslashes and this makes the path not getting returned relative.

Reply:
just tried `FileHelper::normalize`: it will not turn the slashes in the correct form to make `$pathHelper->getRelativePath` return a proper relative path

Reply:
Why wouldn't it? It works everywhere else in PHPStan's codebase.

Reply:
I don't think so.

in other places we also do the `\\ -> /`  dance to make it work:
https://github.com/phpstan/phpstan-src/blob/494b7b0de21c08147a795330d5c233bd0166f841/tests/e2e/ResultCacheEndToEndTest.php#L194-L199

but I found a way to make it work.